tests/operator_test.go

```diff
@@ -2156,7 +2169,37 @@ spec:
 				return true
 			}, 60*time.Second, 1*time.Second).Should(BeTrue())
 
-			patchKvWorkloads(kv.Name, nil)
+			patchKvWorkloads(nil, false, "")
+		})
+
+		It("should reject infra placement configuration with incorrect toleraion operator", func() {
+			incorrectOperator := "foo"
+			incorrectWorkload := v1.ComponentConfig{
+				NodePlacement: &v1.NodePlacement{
+					Tolerations: []k8sv1.Toleration{{
+						Key:      "someKey",
+						Operator: k8sv1.TolerationOperator(incorrectOperator),
+						Value:    "someValue",
+					}},
+				},
+			}
+			errMsg := fmt.Sprintf("Unsupported value: \"%s\"", incorrectOperator)
```

Comment:
Does the exact error message really matters? Also consider this might change at some point (in upstream Kubernetes)...

Reply:
@zcahana Yes, I would like to be precise here in order not to pass the tests on errors other than the one I expect here.